lib/SetAPI/generic/GenericSetNavigator.py

```diff
@@ -284,7 +283,8 @@ def _validate_get_set_items_params(self, params):
 
     def _get_set_info(self, set_refs):
         objects = [
-            build_ws_obj_selector(s["ref"], s.get("path_to_set", [])) for s in set_refs
```

Comment:
!!!!

there's a lot of inconsistency throughout the module over whether this param should be called `path_to_set` or `ref_path_to_set`. Kind of concerning, to say the least!

Reply:
That's more specific and clear, works for me.